assets/survey_themes/fruity/fruityThemeExample.scss

```
@@ -3044,7 +3044,7 @@ ol.breadcrumb.title-bar-breadcrumb {
 
 
 
-/* Define the color for Awesome Bootsrap Checkboxes ; based on bg-primary background-color*/
+/* Define the color for Awesome Bootstrap Checkboxes ; based on bg-primary background-color*/
```

Stylelint (17.6.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)